lib/shell/shellscript.js

```diff
@@ -59,12 +60,18 @@ class ShellScriptWorker {
     }
 }
 
-function prepareErrorFile(directory) {
+async function prepareErrorFile(directory) {
     const errDir = path.resolve(directory, "errors");
-    fs.mkdirSync(errDir, {recursive: true});
+    await fs.mkdirs(errDir, {recursive: true});
     return path.resolve(errDir, "error.json");
 }
 
+async function prepareMimeInfoFile(directory, renditionId) {
+    const mimeDir = path.resolve(directory, "mimeinfo");
+    await fs.mkdirs(mimeDir, {recursive: true});
+    return path.resolve(mimeDir, `mimeinfo-${renditionId}.txt`);
```

Comment:
At this point the shell script worker only supports one rendition at a time, so we don't need to support multiple renditions at a time.

And same "mime" terminology remark.

Suggest to name `type.txt` or `contentype.txt`. And remove this file/the `metadata` folder after each rendition.

Reply:
`type.txt` it is.